### DISCUSSION TITLE
Inspections

### DISCUSSION THREAD

**Discussion by igbjnI05bF** [hargata/lubelog#513] [Ideas]
Love this software so far. The one killer feature that's missing for my use case are inspections with templates. The ability to schedule regular inspections (weekly, bi-weekly, monthly, etc) and then have a template that I can define (check turn signals [pass/fail], check tire pressure [pass/fail], submit inspection with signature on mobile). This video shows a great workflow for this: https://youtu.be/84SyeJ4krZs?si=fDoPeZLMw89LWMnq

The only way I can kinda see this working as-is is to just set recurring reminders for inspections (although it doesn't look like you can schedule more frequently than monthly) and then maybe enter the inspections as service records, but from what I can tell that doesn't allow any templating and doesn't allow any quick check boxes for pass/fail, no signature, etc.

**Comment by hargata** [comment]
We finally got around to this feature, and it will be available in the next release

https://docs.lubelogger.com/Records/Inspections

**Comment by hargata** [comment]
Released, [video walkthrough](https://www.youtube.com/watch?v=_SBe_7bNMuk)
